app/views/feeds/_form.html.erb

```diff
@@ -12,9 +12,9 @@
           <%= render "form_auth", podcast: podcast, feed: feed, form: form %>
         <% end %>
         <%= render "form_audio_format", podcast: podcast, feed: feed, form: form %>
+        <%= render "form_tags", podcast: podcast, feed: feed, form: form %>
         <% if feed.custom? %>
           <%= render "form_ad_zones", podcast: podcast, feed: feed, form: form %>
-          <%= render "form_tags", podcast: podcast, feed: feed, form: form %>
```

Comment:
I moved this out of the `feed.custom` condition so it can appear on the default feed as well.